package.json

```diff
@@ -30,25 +30,6 @@
     "tailwindConfig": "./packages/tailwind-config/tailwind.config.ts"
   },
   "dependencies": {
-    "posthog-js": "^1.203.1"
-  },
-  "//": "Roam Research still uses React 18",
-  "overrides": {
-    "@blueprintjs/core": {
-      "react": "18.2.0",
-      "react-dom": "18.2.0"
-    },
-    "@blueprintjs/datetime": {
-      "react": "18.2.0",
-      "react-dom": "18.2.0"
-    },
-    "@blueprintjs/select": {
-      "react": "18.2.0",
-      "react-dom": "18.2.0"
-    },
-    "roamjs-components": {
-      "react": "18.2.0",
-      "react-dom": "18.2.0"
-    }
+    "posthog-js": "^1.261.8"
```

Comment:
upgraded to newer posthog which has types, unlike older one. BUT it may have undone some of the earlier styling work...

Reply:
Btw forget that, it was a mistake (thinking of birdseatbugs, not posthog)

Reply:
This should move to roam and website package.json anyway.  Minor, can be done later.